UserGuide.md

```diff
@@ -140,6 +140,18 @@ Note that you can not serialize objects with circular references since that will
 // Deserialization
 BagOfPrimitives obj2 = gson.fromJson(json, BagOfPrimitives.class);
 // ==> obj2 is just like obj
+
+// If you the type of your json is an Object you and deserialize into a generic Map class
+Map map = gson.fromJson(json, Map.class);
```

Comment:
Would be better to use `TypeToken` to get a parameterized `Map<..., ...>`. Using raw types is discouraged.
Though the sections "Collections Examples" and "Serializing and Deserializing Generic Types" might already cover that.

Reply:
The point is to be simpler than the exiting example code that uses TypeToken.  A beginning user doesn't need to be confused that.  If you are writing throwaway code or if you don't know or have an exact java class using a plain Map is fine.

Reply:
Though a beginner will then see warnings from the IDE and the compiler and might be confused that something shown in the official user guide causes warnings.

Reply:
IntelliJ gives a lot of spurious warnings.  Beginners will have to get used to ignoring them.

Reply:
I would not call warnings about raw types "spurious". Likely every IDE and `javac` itself warns about it, for a reason. You are basically giving up Java's type safety.

Reply:
Yes when you use Object you give up type safety, but type safety can be an illusion.  What happens if you try to use TypeToken but you get the type wrong?

We are kind of straying from the point.  I want to make it clear to the reader what the simplest way to use Gson is.  If they are good Java developers that like type safety they can take the extra step of using TypeToken or doing something else to ensure type safety.